frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import classes from './AboutResourcePage.module.css';
+import { Select, TextField, TextArea, Button } from '@digdir/design-system-react';
+import { Switch } from 'resourceadm/components/Switch';
+import {
+  resourceSectorsMockOptions,
+  resourceThematicAreaMockOptions,
+  resourceTypeMockOptions,
+} from 'resourceadm/data-mocks/resources';
+
+/**
+ * Page that displays information about a resource
+ */
+export const AboutResourcePage = () => {
+  // TODO - translation
+  // TODO - connect with backend
+
+  const [selectedResource, setSelectedResource] = useState('');
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [homepageURL, setHomepageURL] = useState('');
+  const [keywords, setKeywords] = useState('');
+  const [sectors, setSectors] = useState<string[]>([]);
+  const [thematicArea, setThematicArea] = useState<string[]>([]);
+  const [delegationText, setDelegationText] = useState('');
+  const [showInPublic, setShowInPublic] = useState(false);
+
+  const handleSaveResource = () => {
+    // TODO - Create the resource object
+    // TODO - Split the keyword string into string aray based on the commas
```

Comment:
Will there also be any sort of validation of the data here?

Maybe not a scope of this task, but I noticed a TODO about informing the user that something was optional and that made me think that the things that are not optional will probably need validating.
Maybe add a TODO comment about that together with the other TODO's?

Reply:
Yes, that is correct. There is planned validation here for the future. However, the API has not been made yet and it is unsure what needs validation or not. I added a TODO to make sure that it is remembered at the time of implementation.